webconsole/src/main/java/org/secondbase/webconsole/HttpWebConsole.java

```diff
@@ -0,0 +1,135 @@
+package org.secondbase.webconsole;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.util.ServiceLoader;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.core.moduleconnection.WebConsole;
+import org.secondbase.webconsole.widget.Widget;
+
+/**
+ * A webserver for hosting secondbase servlets using Sun's {@link HttpServer}.
+ */
+public class HttpWebConsole implements SecondBaseModule, WebConsole {
+
+    private static final Logger LOG = Logger.getLogger(HttpWebConsole.class.getName());
+    private final HttpServer server;
+    private int port;
+
+    private final ServiceLoader<Widget> widgets
+            = ServiceLoader.load(Widget.class);
+
+    /**
+     * Basic /health endpoint, returning 200 OK.
+     */
+    private class HealthzHandler implements HttpHandler {
+        public void handle(final HttpExchange t) throws IOException {
+            final byte [] response = "Healthy".getBytes();
+            t.sendResponseHeaders(200, response.length);
+            final OutputStream os = t.getResponseBody();
+            os.write(response);
+            os.close();
+        }
+    }
+
+    /**
+     * Set up the webconsole using port from {@link WebConsoleConfiguration}.
+     * @throws IOException if server can't start on a given port.
+     */
+    public HttpWebConsole() throws IOException {
+        server = HttpServer.create();
+        server.createContext("/healthz", new HealthzHandler());
+    }
+
+    /**
+     * Load WebConsole Flags and set the secondbase webconsole to "this".
```

Comment:
https://www.theguardian.com/media/mind-your-language/2011/may/19/mind-your-language-punctuation-quotations

Wrong and wrong.